airflow/www/views.py

```diff
@@ -635,12 +635,14 @@ class AirflowBaseView(BaseView):
 
     def render_template(self, *args, **kwargs):
         # Add triggerer_job only if we need it
-        if TriggererJob.is_needed():
-            kwargs["triggerer_job"] = lazy_object_proxy.Proxy(TriggererJob.most_recent_job)
+        if TriggererJobRunner.is_needed():
+            kwargs["triggerer_job"] = lazy_object_proxy.Proxy(
+                lambda: most_recent_job(TriggererJobRunner.job_type)
+            )
```

Comment:
```suggestion
            kwargs["triggerer_job"] = lazy_object_proxy.Proxy(TriggererJobRunner.most_revent_job)
```

Can't this use the helper method still?

Reply:
We do not have `most_recent_job` in the BaseJobRunner, but yeah, I think we can add it.

Reply:
Added it back.